src/Sphere.jl

```diff
@@ -169,7 +169,7 @@ end
 Compute the Riemannian [`mean`](@ref mean(M::Manifold, args...)) of `x` using
 [`GeodesicInterpolationWithinRadius`](@ref).
 """
-mean(::Sphere, args...)
+mean(::Sphere, ::Any)
```

Comment:
What's the benefit of `::Any` over `args...`? e.g. if one entered
`?mean(Sphere(2), x, w)`, does it still show this docstring in the REPL?

Reply:
oh, it should be `::Any...` then its just the type of the old `args...` without having an unused variable name (as the first one does not have the `M`).